templates/feed/comment/add-comment.html.twig

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <section>
+        {{ form(form) }}
+    </section>
+{% endblock %}
```

Comment:
Let's have a look at styling these. Take a look at the `templates/auth/registration/register.html.twig` for how to reference the fields set in the FormType and how to add classes to them. The docs have some stuff I think too